modules/routes/main.tf

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/******************************************
+	Routes
+ *****************************************/
+resource "google_compute_route" "route" {
+  for_each = var.routes
+
+  project = var.project_id
+  network = var.network_name
+
+  name                   = each.key
+  description            = lookup(each.value, "description", "")
+  tags                   = compact(split(",", lookup(each.value, "tags", "")))
```

Comment:
Should we change this to `null` if not set?

Reply:
tags needed to stay as `""` for the split function to work. But the other attributes i default to null so the `google_compute_route` resources defaults get used.